internal/app/go-filecoin/node/node.go

```diff
@@ -447,11 +447,11 @@ func (node *Node) setupStorageMining(ctx context.Context) error {
 		PoStProofType: postProofType,
 		SealProofType: sealProofType,
 		Miner:         minerAddr,
-		WorkerThreads: 1,
+		WorkerThreads: 2,
```

Comment:
Sectorbuilder behaves differently depending on whether it's given 1 or more threads. It won't seal if only given 1.